lib/dialogs/keybindings/index.tsx

```diff
@@ -143,7 +143,15 @@ export class AboutDialog extends Component<Props> {
                   <Keys keys={[CmdOrCtrl, 'Shift', 'N']}>Create new note</Keys>
                 </li>
                 <li>
-                  <Keys keys={[CmdOrCtrl, 'Delete']}>Trash note</Keys>
+                  <Keys
+                    keys={
+                      isMacApp
```

Comment:
@belcherj we want to detect macOS from the browser as well. I don't think we care too much about people who are masking their user agent (they probably know what's going on here anyway), though we might lose our user-agent sniffing ability with privacy controls in browser updates